src/book/03-guides/04-general/04-documentation.mdx

```diff
@@ -0,0 +1,88 @@
+---
+section: Guides
+chapter: General
+title: Documentation
+description: How to properly document code.
+slug: /guides/general/documentation
+---
+
+## Conventions
+
+We use the javadoc documentation _style_, all this means is that directives are formatted with a `@`. Note that `@` and `\\` are interchangeable when looking at other sources.
+
+Each declaration, or definition in the absence of declaration, that is a part of an api, e.g. declared in the `.hpp` file, should have a documenting comment.
```

Comment:
This needs to be rewritten, it's too jumbled. What is it you're trying to say here? 'Everything in the `.hpp` should have a documenting comment'?